VSTextField.swift

```diff
@@ -50,6 +53,11 @@ class VSTextField: UITextField {
      A character which will be replaced in formattingPattern by a number
      */
     var secureTextReplacementChar: Character = "\u{25cf}"
+
+    /**
+     True if input number is hexadecimal eg. UUID
+     */
+    var isHexadecimal = false
```

Comment:
I'm not sure if we need to introduce a new state. This can be a computed variable and we can compute its value based on the current state of the ```formatting``` property.